webui/react/src/components/CreateGroupModal.tsx

```diff
@@ -192,20 +190,9 @@ const CreateGroupModalComponent: React.FC<Props> = ({ onClose, users, group }: P
             name={GROUP_NAME_NAME}
             required
             validateTrigger={['onSubmit', 'onChange']}>
-            <Input autoComplete="off" autoFocus maxLength={128} placeholder="Group Name" />
-          </Form.Item>
-          <Form.Item label={USERS_LABEL} name={USERS_NAME}>
-            <Select mode="multiple" optionFilterProp="children" placeholder="Add Users" showSearch>
-              {users
-                ?.filter((u) => u.isActive || currentGroupMembers?.includes(u.id))
-                ?.map((u) => (
-                  <Select.Option key={u.id} value={u.id}>
-                    {getDisplayName(u)}
-                  </Select.Option>
-                ))}
-            </Select>
+            <Input autoComplete="off" autoFocus maxLength={128} placeholder={GROUP_NAME_LABEL} />
```

Comment:
User can still make a group with all whitespace name like `     `. I think we should add `whitespace` rules